app/screens/entry/entry.js

```diff
@@ -122,23 +120,39 @@ export default class Entry extends PureComponent {
         }
 
         if (state.views.root.hydrationComplete) {
+            this.handleHydrationComplete();
+        }
+    };
+
+    handleHydrationComplete = () => {
+        if (this.unsubscribeFromStore) {
```

Comment:
It should only be called once, but I'll make sure to clear `this.unsubscribeFromStore` after it's called so that it doesn't get called twice by mistake